.eslintrc.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  root: true,
+  env: {
+    node: true,
+  },
+  parserOptions: {
+    ecmaVersion: 2021,
+  },
+  plugins: ['only-warn'],
```

Comment:
Can you explain the reason for only using warnings?  These tend to not show up well in github actions, and it's easy to miss them and accidentally merge a PR with warnings, only to notice them later on in a different PR.

Reply:
I don't like errors in IDE from linter, because it's very hard to understand what is not working, it's bad practice that results to missing the real errors.

If you want getting errors in CI, I can fix it using [solution](https://github.com/bfanger/eslint-plugin-only-warn#usage):

```json5
{
  "scripts": {
     "lint": "eslint --max-warnings=0 ...",
   }
}
```

Reply:
Fixed in 49ac41cc.